### DISCUSSION TITLE
I hope to support the ability to copy and paste screenshot images.

### DISCUSSION THREAD

**Discussion by 52lemon6** [rustdesk/rustdesk#8125] [General]
I hope to support the ability to copy and paste screenshot images. Usage scenario: For instance, if I am using a screenshot software (Snipaste) on the controlling device and wish to directly paste it onto the controlled device's WeChat or QQ.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/683
